acceptance/rekor/rekor.go

```diff
@@ -280,17 +273,44 @@ func stubRekorEntryFor(ctx context.Context, data []byte) error {
 		return err
 	}
 
-	dataBase64 := base64.StdEncoding.EncodeToString(data)
+	jsonPath, err := fn(data)
+	if err != nil {
+		return fmt.Errorf("failed to extract JSON path: %w", err)
+	}
 
 	// return this entry for any lookup in Rekor
 	return wiremock.StubFor(ctx, wiremock.Post(wiremock.URLPathEqualTo("/api/v1/log/entries/retrieve")).
-		WithBodyPattern(wiremock.Contains(dataBase64)).
+		WithBodyPattern(wiremock.MatchingJsonPath(jsonPath)).
 		WillReturn(string(body),
 			map[string]string{"Content-Type": "application/json"},
 			200,
 		))
 }
 
+type jsonPathExtractor func([]byte) (string, error)
+
+// jsonPathFromSignature returns the JSON Path expression to be used in the wiremock stub
+// for a signature query. The expression matches the value of the signature's content.
+func jsonPathFromSignature(data []byte) (string, error) {
+	blah := cosign.Signatures{}
```

Comment:
(nit) `blah` looks a bit off
```suggestion
	signatures := cosign.Signatures{}
```

Reply:
oops.. fixed it. Thanks!